diff --git a/src/dcap_provider.cpp b/src/dcap_provider.cpp
index d23c250..34c4d9d 100644
--- a/src/dcap_provider.cpp
+++ b/src/dcap_provider.cpp
@@ -1333,7 +1333,7 @@ static std::string build_tcb_info_url(
         tcb_info_url << base_url;
     }
     else
-        tcb_info_url << get_base_url();
+        tcb_info_url << "https://api.trustedservices.intel.com/sgx/certification";
 
     if (!version.empty())
     {
@@ -1426,7 +1426,7 @@ static std::string build_enclave_id_url(
         qe_id_url << base_url;
     }
     else
-        qe_id_url << get_base_url();
+        qe_id_url << "https://api.trustedservices.intel.com/sgx/certification/";
 
     // Select the correct issuer header name
     if (!version.empty())
@@ -1521,6 +1521,7 @@ static quote3_error_t get_collateral(
                 "Successfully fetched %s from URL: '%s'.",
                 friendly_name.c_str(),
                 url.c_str());
+/*
             std::string cache_control;
             auto get_cache_header_operation = get_unescape_header(*curl_operation, headers::CACHE_CONTROL, &cache_control);
             retval = convert_to_intel_error(get_cache_header_operation);
@@ -1534,6 +1535,7 @@ static quote3_error_t get_collateral(
                     local_cache_add(issuer_chain_cache_name, expiry, issuer_chain.size(), issuer_chain.c_str());
                 }
             }
+*/
         }
 
         return retval;
